src/ui/views/tui_textfield.rs

```diff
@@ -13,11 +17,49 @@ use unicode_width::UnicodeWidthStr;
 use crate::context::AppContext;
 use crate::event::process_event;
 use crate::event::AppEvent;
-use crate::key_command::{complete_command, Command, InteractiveExecute};
+use crate::key_command::constants::COMMANDS;
+use crate::key_command::{Command, Completion, CompletionKind, InteractiveExecute};
 use crate::ui::views::TuiView;
 use crate::ui::widgets::{TuiMenu, TuiMultilineText};
 use crate::ui::AppBackend;
 
+lazy_static! {
+    static ref EXECUTABLES: Vec<&'static str> = {
```

Comment:
This generates the list of binaries in `$PATH`. Should work well both on unix and windows (I tested it on linux, can't confirm about windows).